compiler/testData/codegen/box/operatorConventions/assignOperatorOverload.kt

```diff
@@ -1,5 +1,6 @@
 
 // TARGET_BACKEND: JVM_IR
+// !LANGUAGE:+AssignOperatorOverloadForJvmOldFrontend
```

Comment:
Please also add test with disabled feature to ensure that new behaviour definitely not works if feature is not enabled

Reply:
There is this: [assignOperatorOverloadDisabled.kt](https://github.com/gradle/kotlin/pull/2/files/d74decff7b7a3bda859abd17f7db91fe2c3e522b..a64fca8f58e3166fed6e1d3ddda9bde9b82e06ab#diff-20b915b704de1013df4264283f347d7a1c8e3905a5d8f10f5a6f6c8468e272d5)

Should I do that somehow differently or maybe add more tests? (maybe the name is not the best, maybe assignOperatorOverloadNotEnabled or something else could be better)

Reply:
Or you meant to add additional tests for box testing?

Reply:
Sorry, I didn't notice that test

Naming is fine. I usually name such tests with `_enabled` and `_disabled` suffixes, but it's a question of taste